server/src/http/unleash_client.rs

```diff
@@ -171,6 +172,27 @@ impl UnleashClient {
             )
             .unwrap(),
             custom_headers: Default::default(),
+            service_account_token: Default::default(),
+        }
+    }
+    pub fn from_url_with_service_account_token(
+        server_url: Url,
+        skip_ssl_verification: bool,
+        client_identity: Option<ClientIdentity>,
+        upstream_certificate_file: Option<PathBuf>,
+        service_account_token: String,
+    ) -> Self {
+        Self {
+            urls: UnleashUrls::from_base_url(server_url),
+            backing_client: new_reqwest_client(
+                "unleash_edge".into(),
+                skip_ssl_verification,
+                client_identity,
+                upstream_certificate_file,
+            )
+            .unwrap(),
```

Comment:
Just a double check - we expect this to be a safe unwrap? Or can we swap it for an expect?

Reply:
Yes, it should be, and if it crashes we do want the panic to happen.